pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/manager/config.go

```diff
@@ -104,6 +104,7 @@ spec:
       #             values:
       #               - linux
       securityContext:
+        fsGroup: 65532
```

Comment:
/hold

Why are you adding the fsGroup here? 
It is not required.

Reply:
Described in issue #3016, the controller-pod start got error:
```
1.6656687258729894e+09  ERROR   controller-runtime.client.config        unable to get kubeconfig        {"error": "open /var/run/secrets/kubernetes.io/serviceaccount/token: permission denied"}
```
So add fsGroup (based on [dockerfile.go](https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugins/golang/v3/scaffolds/internal/templates/dockerfile.go#L71)) to resolve it.

Reply:
Maybe I did not make it clear in issue #3016, raised a new issue #3028 to clarify it.
PTAL thanks.

Reply:
Hi @astraw99, 

> Only faced the sa permission error, opened a new issue https://github.com/kubernetes-sigs/kubebuilder/issues/3028 to clarify it.

The default scaffold does have not SA. So, how can that can be faced by default? 
What are the steps required to be performed from scratch so we are able to reproduce this issue?

Reply:
As said in https://github.com/kubernetes-sigs/kubebuilder/issues/3028#issuecomment-1287849547, here add a `fsGroup` comment doc, to make it helpful to resolve the issue.
PTAL thanks.